build/builtin_actors.go

```diff
@@ -0,0 +1,14 @@
+package build
+
+import (
+	"github.com/filecoin-project/lotus/chain/actors"
+)
+
+// TODO a nicer interface would be to embed (and parse) a toml file containing the releases
+var BuiltinActorReleases map[actors.Version]string
```

Comment:
Yes, this would allow for quicker testing without having to recompile.

Reply:
actually, we would embed it so we'd still need a recompile.
It is just slightly nicer.

Reply:
Duh, true. Would like to have that separate `bundles.spec.toml` file though.

Reply:
sure, any syntax suggestions?

Reply:
ok, looks nice enough.

Reply:
In terms of Go structs, I think this would look like (without the toml tags):

```go
type BundleSpec struct {
	Bundles []Bundle
}

type Bundle struct {
	Version actor.Version
	Release string
}
```

Reply:
done.